### PR TITLE
Update samples to kotlinx.coroutines.test 1.6

### PR DESCRIPTION
**Pull request by zsmb13**

### CODE REVIEW

File: WorkManagerMultiprocessSample/app/build.gradle [hunk 1]
```diff
@@ -49,8 +49,6 @@ android {
 }
 
 dependencies {
-
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```
Comment (zsmb13) [inline WorkManagerMultiprocessSample/app/build.gradle:53]: Not required, Kotlin plugin adds it automatically

File: WorkManagerMultiprocessSample/build.gradle [hunk 2]
```diff
@@ -16,25 +16,20 @@
...
-        jcenter()
-    }
+    apply from: 'versions.gradle'
+    addRepos(repositories)
+
     dependencies {
-        classpath "com.android.tools.build:gradle:4.1.3"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath deps.android_gradle_plugin
+        classpath deps.kotlin.plugin
```
Comment (zsmb13) [inline WorkManagerMultiprocessSample/build.gradle:24]: Updated to match the setup used in other projects

File: WorkManagerMultiprocessSample/versions.gradle [hunk 3]
```diff
@@ -22,7 +22,7 @@
 ext.deps = [:]
 def versions = [:]
 versions.activity = '1.1.0'
-versions.android_gradle_plugin = '4.0.0'
+versions.android_gradle_plugin = '4.1.3'
```
Comment (zsmb13) [inline WorkManagerMultiprocessSample/versions.gradle:25]: Overridden manually to not downgrade the version in this project

**Comment by zsmb13** [comment]
@dlam I've bumped the versions across all samples.

**Review by dlam — APPROVED** [review]

**Pull request merged**